README.md

```diff
@@ -20,17 +20,32 @@ accessible URL for these results. For example:
 
 ```json
   "repositories": [
-    "linux-system-roles/network"
+    "linux-system-roles/network",
+    "linux-system-roles/selinux",
+    "linux-system-roles/timesync",
+    "linux-system-roles/tuned",
+    "linux-system-roles/kdump",
+    "linux-system-roles/firewall",
+    "linux-system-roles/postfix",
   ],
   "images": [
     {
-      "name": "fedora-26",
-      "source": "https://download.fedoraproject.org/pub/fedora/linux/releases/26/CloudImages/x86_64/images/Fedora-Cloud-Base-26-1.5.x86_64.qcow2",
+      "name": "fedora-27",
+      "source": "https://download.fedoraproject.org/fedora/linux/releases/27/CloudImages/x86_64/images/Fedora-Atomic-27-1.6.x86_64.qcow2",
```

Comment:
Is it intentional that you used the atomic image here? I think it might make sense to test on atomic, but this was supposed to be the standard fedora image.

Reply:
This was my question as well. Also, the image does not seem to be accessible at that URL.

Reply:
Awesome, you found a bug. I will fix it.